packages/agoric-cli/src/chain-config.js

```diff
@@ -98,7 +98,8 @@ export function finishCosmosApp({
   app['pruning-keep-every'] = '50000';
   app['pruning-interval'] = '1000';
 
-  const apiPort = DEFAULT_API_PORT + (rpcPort - DEFAULT_RPC_PORT) / 100;
+  const apiPort =
+    DEFAULT_API_PORT + Math.ceil((rpcPort - DEFAULT_RPC_PORT) / 100);
```

Comment:
I'm open to other ideas here, but it's important to avoid generating a non-integer port number.

Reply:
This is good.  I have no other suggestions.